terraform/modules/scheduled-job/main.tf

```diff
@@ -17,6 +17,15 @@ terraform {
   }
 }
 
+# Common tags for all resources
+locals {
+  common_tags = merge(var.tags, {
```

Comment:
This seems misnamed -- I'd expect `common_tags` to just be the 3 common tags you list below, not that merged with `var.tags`.  Should it be `all_tags` maybe?

Also, who wins in case of a merge conflict?  I'm guessing that the ones below take precedence over the ones in `var.tags`.  Is that the desired behavior?

Reply:
`all_tags` sounds good to me!